src/Components/Wizard/steps/Pubkeys/NewKeyForm.js

```diff
@@ -95,12 +118,14 @@ const NewSSHKeyForm = ({ setStepValidated }) => {
         <FileUpload
           id="public-key-value"
           onDataChange={handleSSHKeyChange}
-          allowEditingUploadedText={false}
+          allowEditingUploadedText={true}
```

Comment:
nit: you can remove the `true` assignment :)

Reply:
false is default I believe, at least according to docs :)

Reply:
I mean you don't need to specify the truth value in react component:
```js
<Component thruthyProp /> 
// equals 
<Component thruthyProp={true} />
```

Reply:
Ah! now I got it, sure it will look much better :)